atuin-client/src/database.rs

```diff
@@ -286,6 +286,7 @@ impl Database for Sqlite {
         let query = match search_mode {
             SearchMode::Prefix => query,
             SearchMode::FullText => format!("%{}", query),
+            SearchMode::Fuzzy => query.split("").collect::<Vec<&str>>().join("%"),
```

Comment:
The itertools crate (which is already added to this project) provides a `join` method to iterators, without needing to collect them into a vec first: https://docs.rs/itertools/0.10.0/itertools/trait.Itertools.html#method.join

Reply:
An alternative to using `.split("")` too, is to just use `.chars()`

Reply:
thanks for the tip with itertools, that's much nicer

split("") actually isn't equivalent to chars() as it adds an empty string at the beginning and end, which we need

Reply:
Very interesting but also quite unintuitive 🤔 oh well